cmd/runner/resource_class.go

```diff
@@ -94,3 +98,27 @@ func newResourceClassTable(writer io.Writer) *tablewriter.Table {
 func appendResourceClass(table *tablewriter.Table, rc runner.ResourceClass) {
 	table.Append([]string{rc.ResourceClass, rc.Description})
 }
+
+const terms = "If you have not already agreed to Runner Terms in a signed Order, " +
+	"then by continuing to install Runner, " +
+	"you are agreeing to CircleCI's Runner Terms which are found at: https://circleci.com/legal/runner-terms/.\n\n" +
+	"If you already agreed to Runner Terms in a signed Order, " +
+	"the Runner Terms in the signed Order supersede the Runner Terms in the web address above.\n\n" +
+	"If you did not already agree to Runner Terms through a signed Order and do not agree to the Runner Terms in the web address above, " +
+	"please do not install or use Runner.\n\n"
+
+func presentTerms(rc string, o *runnerOpts, cmd *cobra.Command) error {
+	ns, err := o.r.GetNamespaceByResourceClass(rc)
+	if err != nil {
+		return err
+	}
+	rcs, err := o.r.GetResourceClassesByNamespace(ns)
+	if err != nil {
+		return err
+	}
+	// Only present the terms on creation of the first resource class in a namespace
```

Comment:
I thought we had talked about showing this for all RC creations? Or did we settle on doing it once is fine?

Reply:
So the ticket explicitly said the _first_ resource class creation (in a namespace). Asked Steve about it since it involved this bit extra logic, and he said is what was intended.